### DISCUSSION TITLE
Octernship can't apply message

### DISCUSSION THREAD

**Discussion by jayk-gupta** [community/community#52688] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

Maximum application limit reached! You cannot apply to this octernship at this moment.
I had just applied to github student campus expert and rejected in that.
Idk why I can't apply?

**Comment by Ntshangase** [comment]
Hi @jayk-gupta 

It means that particular project is currently closed to new applicants, But you can apply to other projects that are still open.

**Comment by dotandev** [comment]
It depicts that the task has the maximum applications already.
